.github/workflows/tests.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: nightly-2021-05-01
+          toolchain: nightly-2021-11-07
```

Comment:
would be better to introduce [rust-toolchain](https://github.com/galacticcouncil/HydraDX-node/blob/master/rust-toolchain) file, then you dont have to specify it here (use [this](https://github.com/galacticcouncil/Basilisk-node/blob/master/.github/workflows/workflow.yml#L85) action) and it will work for anyone cloning this repo automatically

Reply:
I would do it as separate PR